modules/observe/adoption-insights/proc-configure-adoption-insights.adoc

```diff
@@ -2,24 +2,4 @@
 [id="proc-configure-adoption-insights_{context}"]
 = Configuring the Adoption Insights plugin in {product}
 
-You can enable the Adoption Insights plugin by configuring the {product} Helm chart or the {product} Operator ConfigMap.
-
-.Procedure
-
-* To configure the Adoption Insights plugin in {product-short}, in your {product} `app-config.yaml` file, add the following code:
-+
-.`app-config.yaml` fragment
-[source,terminal]
-----
-app:
-  analytics:
-    adoptionInsights: 
-      maxBufferSize: 20 <1>
-      flushInterval: 5000 <2>
-      debug: false <3>
-      licensedUsers: 2000 <4>
-----
-<1> (Optional) Specifies the maximum buffer size for event batching. The default value is `20`. 
-<2> (Optional) Specifies the flush interval in milliseconds for event batching. The default value is `5000ms`.
-<3> (Optional) The default value is `false`.
-<4> (Optional) Specifies the maximum number of licensed users who can access the RHDH instance. The default value is `100`.
\ No newline at end of file
+The Adoption Insights plugin is preinstalled and enabled on a {product-short} instance by default. You can disable or enable the Adoption Insights plugin and change other parameters, by configuring the {product} Helm chart or the {product} Operator ConfigMap.
```

Comment:
"by configuring the `app-config` ConfigMap, which is created by the {product} Helm chart or Operator."